app/MMB/src/scenes/passenger/PassengerDetail.js

```diff
@@ -77,3 +77,9 @@ export default createRefetchContainer(
     }
   `,
 );
+
+const styles = StyleSheet.create({
+  container: {
+    marginTop: 20,
```

Comment:
Maybe we should add this to the content container instead of setting it on every page?

Reply:
That could definetly be an improvement, but separate PR I guess, since it touches also other screens?

Reply:
I believe it affects only `Layout` component, but OK. 🙂